cli/deployment/api_client.go

```diff
@@ -11,73 +10,245 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// Status represent Deployment status in enum/numerical format
+type Status int
+
+const (
+	Done     Status = 0
```

Comment:
Sorry for not catching all of these. Please add doc comments for exported `const` values as well.